File: cal.java
import java.util.Scanner;

class cal {

    static int cal(int a, char b, int c) {
        switch (b) {
            case '+':
                return (a + c);
            case '-':
                return (a - c);
            case '*':
                return (a * c);
            case '/':
                return (a / c);
            default: 
                return 0;    
        }
    }

    public static void main(String[] arg) {
        System.out.println("Hello.......");
        Scanner sc = new Scanner(System.in);
        int a , c , result = 0;
        char b;
        a = sc.nextInt();
        b = sc.next().charAt(0);
        c = sc.nextInt();
        result = cal(a , b , c);
        System.out.println("------");
        System.out.println(result);
        while(true){
            b = sc.next().charAt(0);
            c = sc.nextInt();
            result = cal(result , b , c);
            System.out.println("------");
            System.out.println(result);

        }

    }
}

File: cal.py
def cal(a , b , c):
    if b=='+':
        return (a+c)
    elif b=='-':
        return (a-c)
    elif b=='*':
        return (a*c)
    elif b=='/':
        return (a/c)
    else:
        return 0
    

print("Hello.......")
a = int(input())
b = input()
c = int(input())
result = cal(a , b , c)
print("------")
print(result)
while True:
    b = input()
    c = int(input())
    result = cal(result , b , c)
    print("------")
    print(result)